AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-table/vho-work-hours-table.component.spec.ts

```diff
@@ -250,13 +250,6 @@ describe('VhoWorkHoursTableComponent', () => {
         fixture.detectChanges();
         expect(component.workHours).toBeNull();
     });
-
-    it('check results input parameter, when wrong type sets to null', () => {
-        component.result = [new VhoNonAvailabilityWorkHoursResponse()];
-        fixture.detectChanges();
-        expect(component.workHours).toBeNull();
-    });
-
     describe('validateTimes', () => {
```

Comment:
Is there a reason we've taken this test out?

Reply:
please ask shaed he suggested to remove this as its not strongly typed.

Reply:
@shaed-parkar - rather than removing tests, perhaps suggesting how to rewrite them would be the better approach, if it's causing an issue,

Reply:
I suggested the test be removed because the test is redundant. The code initially checked for the type but the field is now strongly typed. the check type method doesn't even exist anymore

Reply:
will i mentioned on slack this was removed because it was not strongly typed and causing the failure to build the whole unit test .

Reply:
The component needs to be updated to be strongly typed. see `VhoWorkHoursNonAvailabilityTableComponent` line 38 for the recommended implementation